test/drbgtest.c

```diff
@@ -803,8 +803,14 @@ static void run_multi_thread_test(void)
 
     public = RAND_DRBG_get0_public();
     private = RAND_DRBG_get0_private();
-    RAND_DRBG_set_reseed_time_interval(public, 1);
-    RAND_DRBG_set_reseed_time_interval(private, 1);
+    if (public)
+        RAND_DRBG_set_reseed_time_interval(public, 1);
+    else
+        multi_thread_rand_bytes_succeeded = 0;
+    if (private)
+        RAND_DRBG_set_reseed_time_interval(private, 1);
+    else
+        multi_thread_rand_priv_bytes_succeeded = 0;
```

Comment:
It does not make sense to continue with the multi_thread_test if either of the two DRBGs is not available. Just abort the test instead:

```C
static void run_multi_thread_test(void)
{
    unsigned char buf[256];
    time_t start = time(NULL);
    RAND_DRBG *public, *private;

    if (!TEST_ptr(public = RAND_DRBG_get0_public())
        || !TEST_ptr(private = RAND_DRBG_get0_private())) {
        multi_thread_rand_bytes_succeeded = 0;
        return;
    }
    ...
```